ci-operator/step-registry/openstack/provision/machinesubnet/openstack-provision-machinesubnet-commands.sh

```diff
@@ -107,5 +107,12 @@ if [[ "${CONFIG_TYPE}" == "proxy" || ${OPENSTACK_PROVIDER_NETWORK} != "" ]]; the
   echo "Added bastion subnet ${BASTION_SUBNET_ID} to router: ${BASTION_ROUTER_ID}"
 fi
 
+if [[ "${CONFIG_TYPE}" == "proxy" ]]; then
```

Comment:
Shouldn't the creation of the proxy port belong to the `openstack-provision-bastionproxy` step instead?
It looks like MACHINES_SUBNET_ID and MACHINES_NET_ID are persisted in SHARED_DIR so we have everything we need.